core/templates/dev/head/player/PlayerServices.js

```diff
@@ -356,54 +340,16 @@ oppia.factory('oppiaPlayerService', [
       };
     },
     getExplorationTitle: function() {
-      return _exploration.title;
+      return exploration.title;
     },
     getInteractionHtml: function(stateName, labelForFocusTarget) {
       return oppiaExplorationHtmlFormatterService.getInteractionHtml(
```

Comment:
@BenHenning: is it OK for the Exploration domain object to depend on things like oppiaExplorationHtmlFormatterService? I ask because I wanted to bring getInteractionHtml() into the domain object. I think it's fine, by analogy with exp_domain, but wanted to run it past you.